torch/_inductor/lowering.py

```diff
@@ -4556,7 +4556,7 @@ def reduce_max(x, dim=None, keepdim=False):
     return reduce_amax(x, axis=None, keepdims=keepdim)
 
 
-@register_lowering(aten.min)
+@register_lowering(aten.min, type_promotion_kind=None)
```

Comment:
Will this match the type promotion behavior in eager? I feel like eager might promote

Reply:
Do you have a particular case in mind where promotion would happen? Promotion doesn't happen for the following cases, at least:

```
In [13]: torch.max(torch.zeros((2,2), dtype=torch.bool), dim=1)[0].dtype
Out[13]: torch.bool

In [14]: torch.max(torch.zeros((2,2), dtype=torch.float16), dim=1)[0].dtype
Out[14]: torch.float16

In [15]: torch.max(torch.zeros((2,2), dtype=torch.int32), dim=1)[0].dtype
Out[15]: torch.int32
```